src/OpenLoco/src/Town.h

```diff
@@ -29,6 +30,13 @@ namespace OpenLoco
         metropolis,
     };
 
+    struct Sub497FFCResult
+    {
+        World::Pos3 roadStart;
+        uint16_t tad;
```

Comment:
tad?

Reply:
track and direction. I use tad in a few places.

Reply:
Would it be possible to use a type for this to make it more intuitive?

Reply:
i do actually have a type for it but annoyingly there are like 4-5 variations of it. I can't decide the best approach.